android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -435,6 +435,10 @@ protected void onResume() {
 
         final SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(this);
 
+        id = getIntent().getIntExtra(ConstantStrings.ID, 0);
```

Comment:
The problem is when the app is stopped, so does the lifecycle. Therefore when a list item is clicked, the onCreate method is never called and therefore the sessiondetailsactivity always showed the first clicked item details on any subsequent clicks. So it was necessary to fetch them from the intent in the onresume itself.